### ISSUE TITLE
Add color to layers in the timeline + distinction between groups and layers

### ISSUE THREAD

**Issue by enol** [aseprite/aseprite#1893]
When you are working on files with a huge pile of layers it can be difficult to identify each part of the animation. Adding color to an entire row (or even individual frames) could be really helpful.

The pop-up screen when double-clicking a layer could show a color option with the picker and predefined tones (identical to the one from the frame tag properties).

Also, giving a darker tone to the grouped layers will help to avoid confusion with regular layers.

Attached are two samples: one representing a colored timeline and another that isolates groups from drawing layers.

![aseprite_timeline_color_1](https://user-images.githubusercontent.com/10049471/46581230-61577880-ca35-11e8-8b1c-694ff09fc2e2.jpg)

![aseprite_timeline_color_2](https://user-images.githubusercontent.com/10049471/46581233-65839600-ca35-11e8-9d19-aa0796457df2.jpg)

...
...
...

### Aseprite and System version

* Aseprite version: Aseprite v1.2.9-x64
* System: Windows 10

**Comment by sparr** [comment]
The userdata color setting for layers lets you color them. Is this sufficient for your first request?

![Screenshot from 2020-02-13 22-36-26](https://user-images.githubusercontent.com/121299/74507471-59250e00-4eb1-11ea-9bfc-03485ba95f34.png)

**Comment by enol** [comment]
Yeah! it works fine for me. Only issue I see with it is that is not properly highlighted, but anyway thanks for the tip  ;)

**Comment by enol** [comment]
I’d like to revisit this topic. I’m currently working on a project that relies heavily on layer groups, and it’s becoming increasingly difficult to visually distinguish and manage different layers within complex hierarchies.

I think slightly darkening group rows, combined with preventing cels from being dragged onto them (since they effectively disappear), would be an ideal improvement to the overall Aseprite workflow.
